src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -1448,8 +1540,15 @@ public override IEnumerable<IOperation> Children
         {
             get
             {
-                yield return EventReference;
-                yield return HandlerValue;
+                IOperation current;
+                if ((current = HandlerValue) != null)
+                {
+                    yield return current;
+                }
+                if ((current = EventReference) != null)
```

Comment:
>if ((current = EventReference) != null) [](start = 16, length = 39)

Similar to regular assignments, the ```EventReference``` is evaluated first.

Reply:
done